src/coalton.lisp

```diff
@@ -18,6 +18,10 @@
                                            coalton:define-instance
                                            coalton:repr))
 
+(when (string= (uiop:getenv "COALTON_ENV") "release")
+  (format t "COALTON starging in release mode~%")
```

Comment:
starging -> starting

Reply:
Also do we for sure want it to print something in release mode or would it make more sense to have it only print when coalton starts up in development mode (with some text telling you how to start in release)

Reply:
We could log in development mode, that is what React does. I thought that logging to the repl every time Coalton is loaded sounded kinda annoying.